### DISCUSSION TITLE
Make file not working

### DISCUSSION THREAD

**Discussion by TawfiqAbubaker** [l3af-project/eBPF-Package-Repository#55] [Q&A]
When running ``sudo make all``

I get the following error :

make -C /usr/src/linux/samples/bpf/../../tools/lib/bpf/ RM='rm -rf' EXTRA_CFLAGS="" \
        LDFLAGS= srctree=/usr/src/linux/samples/bpf/../../ O=
make[1]: Entering directory '/usr/src/linux/tools/lib/bpf'
make[1]: Leaving directory '/usr/src/linux/tools/lib/bpf'
make -C /usr/src/linux/ /home/tawfiq/Documents/learning rust/project1/atest/eBPF-Package-Repository/ratelimiting/ BPF_SAMPLES_PATH=/usr/src/linux/samples/bpf
make[1]: Entering directory '/usr/src/linux'
***
*** Configuration file ".config" not found!
***
*** Please run some configurator (e.g. "make oldconfig" or
*** "make menuconfig" or "make xconfig").
***
Makefile:608: include/config/auto.conf: No such file or directory
make[1]: *** No rule to make target '.config', needed by 'include/config/auto.conf'.  Stop.
make[1]: Leaving directory '/usr/src/linux'
make: *** [Makefile:60: build] Error 2

**Comment by sanfern** [comment]
Hi @TawfiqAbubaker, Can you share more details. Kernel version of the host and Kernel source version you are getting this error. Did you run `make defconfig` Building steps [here](https://github.com/l3af-project/l3af-arch/blob/main/dev_environment/provision.sh#L36)
